clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessor.groovy

```diff
@@ -117,10 +130,30 @@ class DefaultOrchestrationProcessor implements OrchestrationProcessor {
       }
     }
 
+    def timedCallable = TimedCallable.forClosure(registry, orchestrationsId, propagate(operationClosure, true))
+    executorService.submit(timedCallable)
+
     task
   }
 
   void autowire(obj) {
     applicationContext.autowireCapableBeanFactory.autowireBean obj
   }
+
+  /**
+   * All thread local variables should be cleared after a thread finishes executing.
+   *
+   * This is particularly important for the inheritable MDC context that is commonly to transmit the auth context.
+   */
+  static void clearThreadLocals() {
+    ["threadLocals", "inheritableThreadLocals"].each {
```

Comment:
Feels like we should probably be doing this in other threaded services ... did some digging and `logback` 1.1.5 no longer sets MDC contexts as inheritable so that would help as well.